config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :movies, only: %i[index show]
+  resources :cinema_halls, only: %i[index show]
```

Comment:
It's not totally related to this pull request, but look into your cinema halls controller file:

```
class CinemaHallsController < ApplicationController
        def index 
            @movies = CinemaHall.all 
            @movies = @movies.where('title like ?', "%#{params[:name]}%") if params[:name]
    
            render json: @movies
        end
    
        def show
            @movies = CinemaHall.find(params[:id])
            render json: @movies
        end 
end
```
Remember to be consistent in naming of variables, just to be sure that it's not copy-paste and you aware of what your doing 👍